src/backend/AbsSecBlock.c

```diff
@@ -409,7 +410,7 @@ int BSL_AbsSecBlock_DecodeFromCBOR(BSL_AbsSecBlock_t *self, BSL_Data_t encoded_c
                 // This is a failure case - should more clearly return?
                 BSL_LOG_ERR("Unhandled case");
                 // NOLINTNEXTLINE
-                exit(1); 
+                exit(1);
```

Comment:
Exit from a library is a big nono. If this cannot be fixed we need a ticket to point to these uses to get them removed.